DockeriseRmap/description.md

```diff
@@ -0,0 +1,59 @@
+# Running the docker image and using it to run the rscript
+
+## Running the docker image
+
+Run with:
+`docker run --rm -v /workspaces/ExpansionHunter/results:/results:ro -v /workspaces/ExpansionHunter/results:/results2:rw -ti <Image ID>`
+
+where results is the location of the current data (will be mentioned in A, C, D and E)
+
+## Running the rscript in the docker container
+
+`cd ../..`
+`bgzip -c results/repeats.vcf > workdir/repeats.vcf.gz`
+`tabix -p vcf workdir/repeats.vcf.gz`
+`touch workdir/annot.hdr`
+`Rscript mapV3.r results/repeats.txt results/repeats_realigned.bam workdir/repeats.vcf results/multi_str.txt workdir/ results2/ repeats`
```

Comment:
Use code fences:

https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks#fenced-code-blocks

````
```
Like this for multiline code
```
````